### PR TITLE
Add demos/instance_occlsegm

### PR DESCRIPTION
**Pull request by wkentaro**
See https://github.com/start-jsk/jsk_apc/tree/instance_occlsegm/demos/instance_occlsegm

### CODE REVIEW

File: demos/instance_occlsegm/README.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+```bash
+# create catkin workspace
+mkdir -p ~/catkin_ws/src
+cd ~/catkin_ws
+git clone https://github.com/start-jsk/jsk_apc.git
+cd src
+
+cd ~/catkin_ws
+source /opt/ros/kinetic/setup.zsh
+catkin build grasp_fusion
```
Comment (pazeshun) [inline demos/instance_occlsegm/README.md]: `grasp_fusion`->`instance_occlsegm`?

File: demos/instance_occlsegm/README.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+catkin build grasp_fusion --no-deps
```
Comment (pazeshun) [inline demos/instance_occlsegm/README.md]: `grasp_fusion`->`instance_occlsegm`?
The previous line is without `--no-deps`. Which is correct?

File: demos/instance_occlsegm/README.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+- <a href="https://drive.google.com/uc?id=1TlaUk9JDDaYAsqrg7TyKZNixKTX-ANIF">20180204_splits.zip</a>
+- <a href="https://drive.google.com/uc?id=1K6GHGyT5uPhq2CbSyoJjt5xppsWXslj_">20180730_splits.zip</a>
+
+
+### With ROS
+
+```bash
+# install python library
+sudo -H pip install git+https://github.com/wkentaro/instance-occlsegm.git
```
Comment (pazeshun) [inline demos/instance_occlsegm/README.md]: Is https://github.com/wkentaro/instance-occlsegm private?
Then it's ok.

File: demos/instance_occlsegm/README.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+# create catkin workspace
+mkdir -p ~/catkin_ws/src
+cd ~/catkin_ws
+git clone https://github.com/wkentaro/instance-occlsegm.git
```
Comment (pazeshun) [inline demos/instance_occlsegm/README.md]: Is https://github.com/wkentaro/instance-occlsegm private?
Then it's ok.

**Comment by wkentaro** [comment]
I fixed the README, but has following errors. Do you have any idea how to fix these?

1. 
https://github.com/start-jsk/jsk_apc/issues/2689

2. 
```
"/home/travis/ros/ws_jsk_apc/src/jsk_apc/demos/instance_occlsegm/ros/instance_occlsegm/python/instance_occlsegm_lib/aug.py", line 3, in <module>
    import imgaug.augmenters as iaa
ImportError: No module named imgaug.augmenters
```

For 2., should we add every dependency to package.xml?

**Comment by pazeshun** [comment]
> "/home/travis/ros/ws_jsk_apc/src/jsk_apc/demos/instance_occlsegm/ros/instance_occlsegm/python/instance_occlsegm_lib/aug.py", line 3, in <module>
>     import imgaug.augmenters as iaa
> ImportError: No module named imgaug.augmenters
> For 2., should we add every dependency to package.xml? (It's a bit pain to send PRs to rosdistro)

Also another error:
```
[instance_occlsegm:make]   File "/home/travis/ros/ws_jsk_apc/src/jsk_apc/demos/instance_occlsegm/ros/instance_occlsegm/tests/data_tests/test_download.py", line 5, in <module>
[instance_occlsegm:make]     import pytest                                                                                
[instance_occlsegm:make] ImportError: No module named pytest
```

`pytest` and `imgaug` seem already added to rosdistro, so I think just adding those dependencies to package.xml is enough:
https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L3235
https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L157

**Comment by wkentaro** [comment]
image

**Comment by wkentaro** [comment]
image

**Pull request merged**